src/main/java/org/example/validators/JobRoleValidator.java

```diff
@@ -0,0 +1,31 @@
+package org.example.validators;
+
+import org.example.models.JobRoleRequest;
+
+public class JobRoleValidator {
+public boolean validateJobRole(
+        final JobRoleRequest jobRoleRequest) throws Exception {
+       final int maxJobNameLength = 100;
+       final int maxLocationLength = 100;
+       final int maxJobDescLength = 100;
+       final int maxJobResponsibilitiesLength = 100;
+       if (jobRoleRequest.getRoleName().length() > maxJobNameLength) {
+           throw new Exception("Role Name greater than 100 characters");
```

Comment:
update all messages to use the variable values

Reply:
resolved